catalog/install.yaml

```diff
@@ -0,0 +1,529 @@
+apiVersion: v1
```

Comment:
I think it makes sense to rename `catalog` to `notifications-catalog` to make directory name more self-explanatory .